Code/Framework/AzToolsFramework/Tests/Input/QtEventToAzInputMapperTests.cpp

```diff
@@ -562,7 +562,7 @@ namespace UnitTest
 
         AZ::Vector2 accumulatedPosition(0.0f,0.0f);
         for(const auto& pos: m_azCursorPositions) {
-            accumulatedPosition += (pos.m_normalizedPositionDelta * AZ::Vector2(WidgetSize.width(), WidgetSize.height()));
+            accumulatedPosition += (pos.m_normalizedPositionDelta * AZ::Vector2((float)WidgetSize.width(), (float)WidgetSize.height()));
```

Comment:
nit: Can you please use `aznumeric_cast<float>(...)` here instead and not old style C casts (I believe they're technically prohibited by the coding standard)